demo/psa/prov-verif-e2e/README.md

```diff
@@ -0,0 +1,152 @@
+# PSA Demonstration Steps
+
+This document explains all the steps one need to follow to successfully run
+end to end PSA Demonstration
+
+## Preconditions
+
+* One need to install `jq` and `curl`.
+
+## Provisioning pipeline setup
+
+* Install `cocli tool` using following command
+
+In a new bourne shell session
+
+```shell
+go install github.com/veraison/corim/cocli@demo-psa-1.0.0
+```
+
+* For provisioning if Supply Chain Endorsements needs to be modified or created fresh, then one needs to use instructions given [here](./COCLI_README.md)
+
+To run provisioning test you need three parallel bourne shell sessions.
+
+In each shell, move to the folder where you are going to clone the GIT repo and do:
```

Comment:
Ok, __really__ nit-picky, but -- "folder" is a desktop metaphor; when talking about command line, it makes more sense to use "directory". 
```suggestion
In each shell, move to the directory where you are going to clone the GIT repo and do:
```